pkg/csi/service/driver.go

```diff
@@ -134,6 +141,61 @@ func (driver *vsphereCSIDriver) BeforeServe(ctx context.Context) error {
 			log.Errorf("failed to read config. Error: %+v", err)
 			return err
 		}
+
+		CSINamespace := os.Getenv(csitypes.EnvVarNamespace)
```

Comment:
Can you create a utility function for lines 145:148 and place it in common or utils package? It will be helpful for future use.

Reply:
done